tests/chocolatey-tests/commands/choco-install.Tests.ps1

```diff
@@ -247,9 +247,11 @@ Describe "choco install" -Tag Chocolatey, InstallCommand {
                 $Output = Invoke-Choco install $env:CHOCOLATEY_TEST_PACKAGES_PATH\alloptions.packages.config --confirm --verbose --debug
 
                 # This is based on two observations: The addition explicitly outputs that it's the Package Configuration.
-                # The configuration output is also about 80 lines.
-                $StartofPackageConfiguration = [array]::IndexOf($Output.Lines, "Package Configuration: CommandName='install'|")
-                $PackageConfigurationOutput = $Output.Lines[$StartofPackageConfiguration..($StartofPackageConfiguration+80)] -join [Environment]::NewLine
+                # The configuration output is about 80 lines.
+                $StartOfPackageConfiguration = [array]::IndexOf($Output.Lines, "Package Configuration Start:")
+                $EndOfPackageConfiguration = [array]::IndexOf($Output.Lines, "Package Configuration End")

```

Comment:
Not so sure about just updating the logic here just to adhere to tests.

Not pushing back against it, just mentioning it feels a bit wrong when that is used as the excuse for the change.

Reply:
Yeah, I'd tend to agree it's not great. However, the alternatives that I could think of would make the tests quite fragile and mean that any configuration change (either adding configuration options, or changing the default settings) could break the tests.